containers/Ideas/IdeaCreate.js

```diff
@@ -247,7 +253,7 @@ export const IdeaCreate = props => {
               <ImageChoiceFormFieldContainer
                 field='Add Image'
                 name='image'
-                onSetImage={(img) => setFieldValue('image', img)}
+                onSetImage={(img) => {setFieldValue('image', img); (img !== null) && setFieldValue('imageChecked', false)}}
```

Comment:
btw would just `!img && setFieldV...` work the same way?

Reply:
Oh yes, img && also works! Changed it, thanks!

Reply:
Yes, that is true, but that is what I meant in the PR description with 'I could not make it work that the validation scheme updates when the image is deleted in the form'. So, if I remove the `img&&`, then when deleting an image, the checkbox gets unchecked, but you are not able so submit, because the validation does not update that there is no image.. maybe try it out!

Reply:
ahh okay, i see! lets go with this, i think it is okay for that case...